source/agora/node/GossipProtocol.d

```diff
@@ -43,37 +47,38 @@ public class GossipProtocol
 
     /***************************************************************************
 
-        If this is the first time this message was received,
+        If this is the first time this transaction was received,
         propagate it to the network.
 
         Params:
-            msg = the received message
+            tx = the received transaction
 
     ***************************************************************************/
 
-    public void receiveMessage (Hash msg) @safe
+    public void receiveTransaction (Transaction tx) @trusted
```

Comment:
Why `@trusted`?

Reply:
Ah leftover from when I was using a custom `hash` which wasn't safe! I'll fix it.